app/assets/stylesheets/style.css

```diff
@@ -75,6 +75,22 @@ h6.dropdown-header {
   border: none;
 }
 
+
+.note-show h1 {
+  font-size: 2rem;
```

Comment:
Hi @illyShelly, sorry for the inconvenience. Could you make this `2.25rem`? Thanks!